storehouse/stock.php

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env php
+
+<?php
+/*
+* Start functions for storehouse
+*/
+
+$carPartsInStock = ['alternator', 'klocki hamulcowe'];
+
+/**
+ * arr (array)  - array to work on
+ * partName (string) - name of part to add to array
+ */
+function addToStock ($arr, $partName) {
+    array_push($arr, $partName);
+    var_dump($arr);
+}
+addToStock($carPartsInStock, 'silnik');
+
+/**
+ * arr (array)  - array to work on
+ */
+function takeFromStock ($arr) {
```

Comment:
To samo co w .js

Funkcja pobierająca z magazanu nie zwraca wartości tylko robi var_dump.